packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -45,62 +40,18 @@ export class CrossmintAASDK extends LoggerWrapper {
             "GET_OR_CREATE_WALLET",
             async () => {
                 try {
-                    if (!isEVMBlockchain(chain)) {
```

Comment:
Should have dropped this when I removed the chain generics [here](https://github.com/Crossmint/crossmint-sdk/pull/491), but this is no longer needed since TS enforces it.